core/pubkey.go

```diff
@@ -10,10 +10,11 @@ type Pubkey []byte
 const pubkeyLength = 128
 
 func NewPubkey(p [4]*big.Int) Pubkey {
```

Comment:
@kilic can I request a quick review of this function?

Reply:
Looks about right